src/basic/unit-def.c

```diff
@@ -162,6 +162,7 @@ DEFINE_STRING_TABLE_LOOKUP(scope_state, ScopeState);
 
 static const char* const service_state_table[_SERVICE_STATE_MAX] = {
         [SERVICE_DEAD] = "dead",
+        [SERVICE_CONDITION] = "exec-condition",
```

Comment:
the string version and the C name of an enum value should always remain in sync, i.e. the string should be changed to `"condition"`i figure